src/scalarstats.jl

```diff
@@ -166,16 +206,40 @@ nquantile{T<:Real}(v::AbstractArray{T}, n::Integer) = quantile(v, (0:n)/n)
 #############################
 
 # span, i.e. the range minimum(x):maximum(x)
+"""
+    span(x) -> Range
+
+Return the span of an integer array, i.e. the range `minimum(x):maximum(x)`.
+"""
 span{T<:Integer}(x::AbstractArray{T}) = ((a, b) = extrema(x); a:b)
 
 # Variation coefficient: std / mean
+"""
+    variation(x[, m]) -> Float
+
+Return the coefficient of variation of an array `x`, optionally specifying
+a precomputed mean `m`. The coefficient of variation is the ratio of the
+standard deviation to the mean.
+"""
 variation{T<:Real}(x::AbstractArray{T}, m::Real) = stdm(x, m) / m
 variation{T<:Real}(x::AbstractArray{T}) = variation(x, mean(x))
 
 # Standard error of the mean: std(a) / sqrt(len)
+"""
+    sem(a) -> Float
+
+Return the standard error of the mean of `a`.
+"""
 sem{T<:Real}(a::AbstractArray{T}) = sqrt(var(a) / length(a))
 
 # Median absolute deviation
+"""
+    mad(v, [center,] [constant=1.4826]) -> Float
+
+Compute the median absolute deviation of `v`, optionally specifying a
+precomputed median `center`. The function accepts a keyword argument
```

Comment:
@nalimilan Full disclosure I have no idea what that scaling factor is for or why that's the default.